model/docs/example_with_datasets.qmd

```diff
@@ -260,6 +259,43 @@ out = hosp_model.plot_posterior(
 )
 ```
 
+Test `posterior_predictive` and `prior_predictive` methods
+
+```{python}
+# | label: fig-demonstrate-use-of-predictive-methods
+# | fig-cap: Hospital Admissions posterior distribution with plot_lm
+import arviz as az
+
+idata = az.from_numpyro(
+    hosp_model.mcmc,
+    posterior_predictive=hosp_model.posterior_predictive(
+        n_timepoints_to_simulate=len(dat["daily_hosp_admits"])
+    ),
+    prior=hosp_model.prior_predictive(
+        n_timepoints_to_simulate=len(dat["daily_hosp_admits"]),
+        numpyro_predictive_args={"num_samples": 1000},
+    ),
+)
+
+az.plot_lm(
+    "negbinom_rv",
+    idata=idata,
+);
+```
+
+We can also use the `plot_ppc` method to compare the posterior predictive samples with the observed data.
+
+```{python}
```

Comment:
Can you explain what this plot is showing?

Reply:
I still don't understand this plot. Why is are the posterior predictive dots so different from the posterior predictive mean?